tests/test_mission_phase_race_condition.py

```
@@ -0,0 +1,200 @@
+        results = []
+        errors = []
+        
+        def slow_transition():
+            """Simulate a slow transition."""
+            try:
+                # This will hold the lock
+                state_machine._is_transitioning = True
+                time.sleep(0.1)  # Simulate long transition
+                state_machine._is_transitioning = False
+                results.append("completed")
+            except Exception as e:
```

Ruff (0.14.10)

Do not catch blind exception: `Exception`

(BLE001)